src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/NocRespondentRepresentativeService.java

```diff
@@ -137,8 +142,14 @@ public void updateRepresentativesAccess(CallbackRequest callbackRequest) throws
 
     }
 
-    public List<ChangeOrganisationRequest> getRepresentationChanges(CaseData  after,
-                                                                    CaseData before) {
+    /**
+     * Identifies differences in representation.
+     * @param after - case data after event
+     * @param before - case data before event was triggered
+     * @return list of change organisation requests for any changes detected
+     */
+    public List<ChangeOrganisationRequest> identifyRepresentationChanges(CaseData  after,
```

Comment:
double space here between CaseData and after. I'm surprised checkstyle didnt flag this